src/components/List/List.tsx

```diff
@@ -11,35 +12,87 @@ export interface ListProps {
   id?: string;
   /** Modifiers array for use with BEM. See how to work with modifiers and BEM here: http://getbem.com/introduction/ */
   modifiers?: any[];
-  /** Ordered or Unordered */
+  /** Ordered, Unordered, or Definition */
   type: ListTypes;
+  /** An optional title that will appear over the list (only applies to Definition Lists) */
+  title?: string;
 }
 
 export default function List(props: React.PropsWithChildren<ListProps>) {
-  const { blockName, children, className, id, modifiers = [], type } = props;
+  const {
+    blockName,
+    children,
+    className,
+    id,
+    modifiers = [],
+    type,
+    title,
+  } = props;
 
   const baseClass = "list";
 
   let listTag;
+  let errorText = "";
 
-  React.Children.map(children, function (child: React.ReactElement) {
-    if (child.type !== "li") {
-      throw new Error("Direct children of `List` should be `<li>`s");
-    }
-  });
-
-  if (type === ListTypes.Unordered) {
-    listTag = (
-      <ul id={id} className={bem(baseClass, modifiers, blockName, [className])}>
-        {children}
-      </ul>
-    );
-  } else if (type === ListTypes.Ordered) {
-    listTag = (
-      <ol id={id} className={bem(baseClass, modifiers, blockName, [className])}>
-        {children}
-      </ol>
-    );
+  switch (type) {
+    case ListTypes.Ordered:
+      errorText = "Direct children of `List` (ordered) should be `<li>`s";
+      React.Children.map(children, function (child: React.ReactElement) {
+        if (child.type !== "li") {
+          throw new Error(errorText);
+        }
+      });
+      listTag = (
+        <ol
+          id={id}
+          className={bem(baseClass, modifiers, blockName, [className])}
+        >
+          {children}
+        </ol>
+      );
+      break;
+    case ListTypes.Unordered:
+      errorText = "Direct children of `List` (unordered) should be `<li>`s";
+      React.Children.map(children, function (child: React.ReactElement) {
+        if (child.type !== "li") {
+          throw new Error(errorText);
+        }
+      });
+      listTag = (
+        <ul
+          id={id}
+          className={bem(baseClass, modifiers, blockName, [className])}
+        >
+          {children}
+        </ul>
+      );
+      break;
+    case ListTypes.Definition:
+      errorText =
+        "Direct children of `List` (definition) should be `<dt>`s or `<dd>`s";
+      React.Children.map(children, function (child: React.ReactElement) {
+        if (
+          child.type !== "dt" &&
+          child.type !== "dd" &&
+          child.type !== React.Fragment
+        ) {
+          throw new Error(errorText);
+        }
+      });
+      listTag = (
+        <section
+          id={id}
+          className={bem("definition-list", modifiers, baseClass)}
+        >
+          {title && (
+            <Heading id="heading1" level={3}>
```

Comment:
can we better ensure uniqueness of this heading id, maybe by using the id of the list?

Reply:
will DL titles always be pure strings?  (eg: will they never have icons or parts of it be `<em>`)?)

Reply:
> will DL titles always be pure strings? (eg: will they never have icons or parts of it be `<em>`)?)

Yes they will only be strings.  If there becomes a need to support other markup, that will need to be specified from the UX/Design team, added to the Figma, etc...